### PR TITLE
SetVisible Sources, fixes #190

### LINKED ISSUE #109: New UI bug(s)
Hi,

- load a bunch of xml/h5
- show sources
- projection mode: Average
- adjust contrast, adjust view
- open new UI (I like it!)
- click the source/group icon at the top

``` 
BdvHandlePostprocessor:BdvHandle found.
Exception in thread "PainterThread" java.lang.ArrayIndexOutOfBoundsException: -1
	at java.util.ArrayList.elementData(ArrayList.java:422)
	at java.util.ArrayList.get(ArrayList.java:435)
	at bdv.viewer.render.MultiResolutionRenderer.createProjector(MultiResolutionRenderer.java:652)
	at bdv.viewer.render.MultiResolutionRenderer.paint(MultiResolutionRenderer.java:511)
	at bdv.viewer.ViewerPanel.paint(ViewerPanel.java:486)
	at net.imglib2.ui.PainterThread.run(PainterThread.java:86)
```

### LINKED ISSUE #190: SetVisible API change
```
public void makeInvisible(SourceAndConverter sac) {
        getDisplaysOf(sac).forEach(bdvhr -> bdvhr.getViewerPanel().state().setSourceActive(sac, false));
    }
```

We could do it like below to (i) save one function and (b) make it a bit more convenient to use, and (c) make it more consistent how it is in the bdv API:
```
public void setVisible(SourceAndConverter sac, boolean visible) {
        getDisplaysOf(sac).forEach(bdvhr -> bdvhr.getViewerPanel().state().setSourceActive(sac, boolean));
    }
```

### PR DESCRIPTION
**Pull request by tischi**

### CODE REVIEW

**Comment by tischi** [comment]
@NicoKiaru 

Sorry! I mistyped the issue!

This one address #190 (and not #109).

**Comment by tischi** [comment]
I managed to change the message of the commit to now address the correct issue.

File: src/main/java/sc/fiji/bdvpg/scijava/services/SourceAndConverterBdvDisplayService.java [hunk 1]
```diff
@@ -161,10 +161,20 @@ public void show(SourceAndConverter... sacs) {
      * Makes visible a source, makes it visible in all bdvs according to BdvhReferences
      * @param sac source to display
      */
+    @Deprecated // Use setVisible instead
```
Comment (NicoKiaru) [inline src/main/java/sc/fiji/bdvpg/scijava/services/SourceAndConverterBdvDisplayService.java]: Nice!

**Review by NicoKiaru — COMMENTED** [review]
I'll get rid of the deprecated methods then merge

**Comment by tischi** [comment]
> I'll get rid of the deprecated methods then merge

I did it on purpose like this, because I was not sure whether your downstream repositories would break if I just removed them... 😉

**Comment by NicoKiaru** [comment]
Great idea, really appreciated! It s early enough and not too complicated to change

**Pull request merged**